### ISSUE TITLE
Latest version fails to launch.

### ISSUE THREAD

**Issue by not-chicken** [sharkwouter/minigalaxy#143]
As reported [here](https://github.com/void-linux/void-packages/issues/21347) minigalaxy 0.9.4 fails to launch stating 
```
gi.repository.GLib.Error: g-io-error-quark: Error opening file /share/minigalaxy/ui/login.ui: No such file or directory.
```
This seems to be caused by some error implementing relative paths by this [commit](https://github.com/sharkwouter/minigalaxy/commit/b0f71e5c05972314bf5a7318019e600b2b8d9daf).

I tried but couldn't find out what's wrong on my own. 
Perhaps this is relevant https://stackoverflow.com/a/1945930.

**Comment by Linuxuser437442** [comment]
Same here on  5.6.7-1-MANJARO
Looks like it should be calling /usr/share/..... instead of /share/....

**Comment by not-chicken** [comment]
@Linuxuser437442 Could you please test with this patch?

```diff
diff --git a/minigalaxy/paths.py b/minigalaxy/paths.py
index 16dd48d..2884dc6 100644
--- a/minigalaxy/paths.py
+++ b/minigalaxy/paths.py
@@ -1,27 +1,30 @@
 import os
 import sys
+from pathlib import Path
 
-LAUNCH_DIR = os.path.abspath(os.path.dirname(sys.argv[0]))
+LAUNCH_DIR = Path(sys.argv[0]).parent.resolve()
 
-CONFIG_DIR = os.path.join(os.getenv('XDG_CONFIG_HOME', os.path.expanduser('~/.config')), "minigalaxy")
-CONFIG_FILE_PATH = os.path.join(CONFIG_DIR, "config.json")
-CACHE_DIR = os.path.join(os.getenv('XDG_CACHE_HOME', os.path.expanduser('~/.cache')), "minigalaxy")
+CONFIG_DIR = Path(os.getenv('XDG_CONFIG_HOME', Path('~/.config').expanduser()) / 'minigalaxy')
+CONFIG_FILE_PATH = str(CONFIG_DIR / 'config.json')
+CACHE_DIR = Path(os.getenv('XDG_CACHE_HOME', Path('~/.cache').expanduser()) / 'minigalaxy')
 
-THUMBNAIL_DIR = os.path.join(CACHE_DIR, "thumbnails")
-DEFAULT_INSTALL_DIR = os.path.expanduser("~/GOG Games")
+THUMBNAIL_DIR = CACHE_DIR / 'thumbnails'
+DEFAULT_INSTALL_DIR = Path('~/GOG Games').expanduser()
 
-UI_DIR = os.path.abspath(os.path.join(LAUNCH_DIR, "../data/ui"))
-if not os.path.exists(UI_DIR):
-    UI_DIR = os.path.abspath(os.path.join(LAUNCH_DIR, "../share/minigalaxy/ui"))
+UI_DIR = LAUNCH_DIR / '..' / 'data' / 'ui'
+if not UI_DIR.exists():
+    UI_DIR = LAUNCH_DIR / '..' / 'share' / 'minigalaxy' / 'ui'
 
-LOGO_IMAGE_PATH = os.path.abspath(os.path.join(LAUNCH_DIR, "../data/icons/192x192/io.github.sharkwouter.Minigalaxy.png"))
-if not os.path.exists(LOGO_IMAGE_PATH):
-    LOGO_IMAGE_PATH = os.path.abspath(os.path.join(LAUNCH_DIR, "../share/icons/hicolor/192x192/apps/io.github.sharkwouter.Minigalaxy.png"))
+LOGO_IMAGE_PATH = LAUNCH_DIR / '..' / 'data' / 'icons' / '192x192' / 'io.github.sharkwouter.Minigalaxy.png'
+if not LOGO_IMAGE_PATH.exists():
+    LOGO_IMAGE_PATH = LAUNCH_DIR / '..' / 'share' / 'icons'/ 'hicolor' / '192x192' / 'apps' / 'io.github.sharkwouter.Minigalaxy.png'
+LOGO_IMAGE_PATH = str(LOGO_IMAGE_PATH)
 
-ICON_WINE_PATH = os.path.abspath(os.path.join(LAUNCH_DIR, "../data/images/winehq_logo_glass.png"))
-if not os.path.exists(ICON_WINE_PATH):
-    ICON_WINE_PATH = os.path.abspath(os.path.join(LAUNCH_DIR, "../share/minigalaxy/images/winehq_logo_glass.png"))
+ICON_WINE_PATH = LAUNCH_DIR/'..'/'data'/'images'/'winehq_logo_glass.png'
+if not ICON_WINE_PATH.exists():
+    ICON_WINE_PATH = LAUNCH_DIR / '..' / 'share' / 'minigalaxy' / 'images' / 'winehq_logo_glass.png'
+ICON_WINE_PATH = str(ICON_WINE_PATH)
 
-LOCALE_DIR = os.path.abspath(os.path.join(LAUNCH_DIR, "../data/mo"))
-if not os.path.exists(LOCALE_DIR):
-    LOCALE_DIR = os.path.abspath(os.path.join(LAUNCH_DIR, "../share/minigalaxy/translations"))
+LOCALE_DIR = LAUNCH_DIR / '..' / 'data' / 'mo'
+if not LOCALE_DIR.exists:
+    LOCALE_DIR = LAUNCH_DIR / '..' / 'share' / 'minigalaxy' / 'translations'
```

**Comment by sharkwouter** [comment]
Thanks for the report. In Manjaro (and probably Void Linux too) ``/bin`` and ``/usr/bin`` are the same directory, so maybe the ``minigalaxy`` command will be run as ``/bin/minigalaxy``, which wouldn't work. I'm not sure why this is not a problem on Arch though. Could you try to see what happens if you run ``PATH=/usr/bin:$PATH minigalaxy``? Then I'll know for sure that's the cause.

**Comment by sharkwouter** [comment]
Could you guys test the bugfix/143 branch? I've commit a potential fix to it.

**Comment by not-chicken** [comment]
@sharkwouter  Yes, `/bin` is a symlink to `/usr/bin` in Void, minigalaxy indeed was being run as `/bin/minigalaxy`:
```
[~]$ echo $PATH
/usr/local/bin:/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/sbin
[~]$ type minigalaxy 
minigalaxy is hashed (/bin/minigalaxy)
```
That commit does work around this case where `/bin/` is before `/usr/bin` in path. Thanks!. 
Arch be might be having it the other way around and hence it worked there.
Though I still wonder if we could some up with a more robust solution, which wouldn't be effected by order of directories in path.

**Comment by not-chicken** [comment]
resolved by #149
